docs/how-to-get-help.md

```diff
@@ -24,9 +24,9 @@ then your second port of call should be to ask your question in any channel to w
 (Your first port of call should be to ask your question in the [Q&A forum](#qa-forum).)
 
 You may wish to direct your question to the **tech support team**.
-To do so, include the string `tech-support` in your question
+To do so, include the string `tech-support` (that's with a dash and no spaces) in your question
```

Comment:
😆 this string literal, by which I mean this string literal

Reply:
I literally mean literally this literal string